fr-FR.json

```diff
@@ -1,9 +1,9 @@
 {
   "1 {0} = {1} {2}": "1 {0} = {1} {2}",
   "24H PNL": "24H PNL",
-  "A Launch Bowl<br></br>brought to you by Cats,<br></br>for everyone": "A Launch Bowl<br></br>brought to you by Cats,<br></br>for everyone",
+  "A Launch Bowl<br></br>brought to you by Cats,<br></br>for everyone": "Un bol de lancement présenté par les chats,pour tout le monde",
```

Comment:
@yoann54 Could you please use `<br></br>` tag here? It's look better when we break line.

Reply:
oh ok sorry i didn't see your message before...